include/regs.h

```diff
@@ -99,7 +99,7 @@
 #define R_TEXTBOX_HEIGHT                         YREG(23)
 #define R_TEXTBOX_ICON_XPOS                      YREG(71)
 #define R_TEXTBOX_ICON_YPOS                      YREG(72)
-#define R_TEXTBOX_ICON_SIZE                      YREG(75)
+#define R_TEXTBOX_ICON_DIMENSIONS                YREG(75)
```

Comment:
this one was confusing at a glance, cause its called "dimensions" but its just one reg. After looking at the code i see what you mean now
not sure if theres a better name to convey that, or just needs a clarifying comment that mentions that its a context-dependent single dimension

Reply:
"side length" ?
`R_TEXTBOX_ICON_SIDE_LENGTH` ?
is that any better?

Reply:
maybe? then its kinda just a long winded way of saying dimension
I guess its the plural that confused me when its just a single number. it might make more sense to name it singular, and then you can see from the code that its used for both length and width

if its just a me problem tho then can just leave it